scrapy/core/downloader/__init__.py

```diff
@@ -6,7 +6,16 @@
 from collections import deque
 
 import six
-from twisted.internet import reactor, defer, task
+import asyncio
+'''
+from twisted.internet import asyncioreactor
+try:
+    asyncioreactor.install(asyncio.get_event_loop())
+except Exception:
+    pass
+'''
```

Comment:
why are you importing `asyncio` in many different places? If it needs to be imported before anything else please put it on `scrapy/__init__.py` only.